kubernetes/samples/scripts/create-weblogic-domain/model-in-image/utils/wl-pod-wait.sh

```diff
@@ -16,23 +16,31 @@ usage() {
   Usage:
 
     $(basename $0) [-n mynamespace] [-d mydomainuid] \\
-       [-p expected_pod_count] [-t timeout_secs] [-q]
+       [-t timeout_secs] [-q]
```

Comment:
should we include [-p 0 | Completed] in usage?

Reply:
good catch! fixed